### PR TITLE
Expose Metacritic Fields On AppInfo

### PR DESCRIPTION
**Pull request by AbdulDridi**
## Checklist

- [x] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: steam/manifest.py [hunk 1]
```diff
@@ -967,6 +970,13 @@ def __init__(
         """This app's review score."""
         self.review_percentage = int(common.get("review_percentage", 0))
         """This app's review percentage."""
+        self.metacritic_name = common.get("metacritic_name", None)
+        """This app's metacritic name."""
+        self.metacritic_score = int(common.get("metacritic_score")) if "metacritic_score" in common else None
+        """This app's metacritic score."""
+        self.metacritic_fullurl = common.get("metacritic_fullurl", None)
```
Comment (Gobot1234) [inline steam/manifest.py:977]: Just url would suffice

**Review by Gobot1234 — CHANGES_REQUESTED** [review]
Can you also update types/manifest so that these changes are typed

File: steam/manifest.py (hunk 1 above)
Comment (AbdulDridi) [reply steam/manifest.py:977]: Is this not it? 

https://github.com/Gobot1234/steam.py/blob/b4a0bc7270691a84aadd684429e47dd7ed3c6e29/steam/types/manifest.py#L61

Comment (AbdulDridi) [reply steam/manifest.py:977]: ah, I should change the score to int

Comment (Gobot1234) [reply steam/manifest.py:977]: Change it to VDFInt cause it is still technically a string

Comment (AbdulDridi) [reply steam/manifest.py:977]: Done

**Comment by Gobot1234** [comment]
Thanks

**Pull request merged**